chidioguejiofor/Registration-Form
# #158432703: User should insert work history

### What does this PR do?
- adds work history page to the app
- adds education page to the app
- improves the structure of the web form
- confirms that form looks good on all devices

### What are the Tasks to be completed?
- user should be provide work and education history

###  Any Background Context?
-  None

### Credits
- None

### How can this be tested ? 
- Once you go to the gh-pages link and navigate to education page you would see a form that looks like this 
![education](https://user-images.githubusercontent.com/38565349/41627747-d5bdefe8-7421-11e8-944d-1394844fe3f2.PNG)
![education 2](https://user-images.githubusercontent.com/38565349/41627752-d91d15e2-7421-11e8-8278-b6502a018042.PNG)
- The experience page looks like this
![experience](https://user-images.githubusercontent.com/38565349/41627786-0f99ff86-7422-11e8-9cc8-9869df9a90d2.PNG)
![experience 2](https://user-images.githubusercontent.com/38565349/41627789-13e0c340-7422-11e8-8587-70b482f7b04a.PNG)

### Relevant PT Stories
- 158432929 
- 158432703
